packages/next/bin/next.ts

```diff
@@ -122,6 +126,10 @@ commands[command]()
     }
   })
 
+const config = traceFn(tracer.startSpan('load-next-config'), () =>
+  loadConfig(PHASE_PRODUCTION_BUILD, process.cwd())
```

Comment:
This will likely cause issues down the road as we'll want to move back to having multiple processes being managed in order to reboot on config changes.

Reply:
It also causes double-execution of the config file in it's current form

Reply:
Noted, thanks. Definitely not worth having that message in there with those concerns :)

Removed.